account_payment_show_invoice/__manifest__.py

```diff
@@ -5,7 +5,7 @@
     "name": "Account Payment Show Invoice",
     "summary": "Extends the tree view of payments to show the paid invoices "
                "related to the payments using the vendor reference by default",
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.1.0",
     "category": "Account-payment",
     "website": "https://odoo-community.org/",
     "author": "Eficent, Odoo Community Association (OCA)",
```

Comment:
@rousseldenis The new field does not appear. You forgot to add the view file in the manifest.

Reply:
Done